app/services/selection/selection.ts

```diff
@@ -548,6 +550,15 @@ export class Selection implements ISelection {
     this.getItems().forEach(item => item.resetTransform());
   }
 
+  scale(scale: IVec2, origin: IVec2 = AnchorPositions[AnchorPoint.Center]) {
+    const originPos = this.getBoundingRect().getOffsetFromOrigin(origin);
+    this.getItems().forEach(item => item.scaleWithOffset(scale, originPos));
+  }
+
+  scaleWithOffset(scale: IVec2, offset: IVec2) {
```

Comment:
Not sure I picked the best name to explain this method. It scale the items and use an offset in x,y coords as an anchor point